### DISCUSSION TITLE
Student benefits troubles.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#138581] [GitHub Education] [answered]
### Hello,

A few months ago (in May I think) I submitted an application for the Student Benefits program, and it seems that the AI model assessing the image did not detect information, I tried multiple documents and all of them said that the name of the school was missing even though it was definitely there, black and white. So I was wondering if there was a way to fix this? I don't really have access to any more documents I haven't tried and I already tried multiple photos in different lights. If you are wondering why I am only making a discussion now is I just gave up in May and today wanted to try again.

Thanks, 
Callum

**Comment by ghost** [comment]
Okay, update, I took a photo of my ID again and now it says it is accepted but I don't have the benefits yet which I guess is a step in the right direction?

**Reply by davevad93** [reply, marked as answer]
Hi @fenziie ,

If your application has been approved your status will be "`Congrats! Your request for @fenziie was approved and your benefits will be available soon.`" which means you have to wait. The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @fenziie was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ghost** [reply]
Thanks :*)
